mobify/platform-scaffold
# Suppress output from Lighthouse 😶

## Changes
- Use `--quiet` flag and suppress unrelated warnings from Lighthouse test run

## How to test-drive this PR
- Browse to https://circleci.com/gh/mobify/platform-scaffold/3044?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
- Expand run-all-tests-on-ci.sh step and notice how succinct it is 👏
